test/test_database.py

```diff
@@ -0,0 +1,99 @@
+from paste.fixture import TestApp
+from nose.tools import *
+from components import model
+import os
+import psycopg2
+
+# HOW TO RUN NOSE TESTS
+# 1. Make sure you are in cs-modify main directory
+# 2. Make sure the path "C:\Python27\Scripts" is added in your environment variables
+# 3. Enter in cmd: "nosetests test/"
+# 4. Nose will run all the tests inside the test/ folder
+
+class TestCode():
+    def __init__(self):
+        self.moduleTableColumnCount = 5
+        self.testModuleCode = "CS2103T"
+        self.testModuleName = "Software Engineering"
+        self.testModuleDesc = "This module introduces the necessary conceptual and analytical tools for systematic and rigorous development of software systems."
+        self.testModuleMC = 4
+        self.testModuleStatus = "Active"
+
+        self.mountingTableColumnCount = 3
+        self.testModuleMountedCount = 2
+        self.testModuleFixedMountedAySem1 = "AY 16/17 Sem 1"
+        self.testModuleFixedMountedAySem2 = "AY 16/17 Sem 2"
+        self.testModuleTentativeMountedAySem1 = "AY 17/18 Sem 1"
+        self.testModuleTentativeMountedAySem2 = "AY 17/18 Sem 2"
+        self.testModuleQuota1 = 190
+        self.testModuleQuota2 = 160
+    
+    def test_getModule(self):
```

Comment:
Function naming convention needs to be changed to test_<something>_<something>...